website/docs/r/database_user.html.markdown

```diff
@@ -98,6 +98,11 @@ Accepted values include:
   * `USER` - New database user has AWS IAM user credentials.
   * `ROLE` -  New database user has credentials associated with an AWS IAM role.
 
+* `ldap_auth_type` - (Optional) Method by which the provided username is authenticated. If no value is given, Atlas uses the default value of NONE.
+  * `NONE` -	Atlas authenticates this user through SCRAM-SHA, not LDAP.
```

Comment:
please include the SCRAM-SHA link from the source docs: https://docs.mongodb.com/manual/core/security-scram